src/scenes/Partners/Detail/Tabs/Projects/PartnerDetailProjects.tsx

```diff
@@ -0,0 +1,33 @@
+import { Grid, Typography } from '@mui/material';
+import { PartnerDetailsFragment } from '../../PartnerDetail.graphql';
+import { PartnerDetailProjectsTable } from './PartnerDetailProjectsTable';
+
+interface Props {
+  partner: PartnerDetailsFragment | undefined;
+}
+
+export const PartnerDetailProjects = ({ partner }: Props) => {
```

Comment:
Now that this is on a different view, it might make more sense performance wise to not fetch this list until this tab is activated. This could have it's own query to grab that list. We can always circle back to that.

Reply:
Wouldn't this mean we also have to fetch the data for each tab when going back to a previous tab or cache the data after first render?